app/src/main/java/edu/grinnell/kdic/RadioService.java

```diff
@@ -225,21 +200,21 @@ public void setRunOnStreamPrepared(final Runnable runOnStreamPrepared) {
         this.runOnStreamPrepared = runOnStreamPrepared;
     }
 
+    /*
+        Plays the currently loaded stream. If stream is not loaded, load stream and play.
+    */
     public void play() {
         if (isLoaded) { // if stream is loaded
 
             timer.cancel(); // cancel the stop timer if it is loaded
```

Comment:
These lines are self-explanatory, so the inline comments are just visual clutter.